### PR TITLE
Update Overview section

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-2625](https://team-1624093970686.atlassian.net/browse/AR-2625).

## Changes

- Add overview feature icons
- Update overview section code
- Add radial gradient to icons

## Screenshots

Screenshot from 2022-05-26 16-59-00
Screenshot from 2022-05-26 16-59-20

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: components/OverviewSection.vue [hunk 1]
```diff
@@ -1,118 +1,113 @@
...
+          justify="space-between"
+          align="center"
+          sm-align="start"
+          gap="2rem"
+        >
+          <div class="icon-container">
+            <v-image path="images/arcana-overview-auth.svg" alt="Arcana Auth" />
+          </div>
+          <div class="overview-feature">
+            <div>
```
Comment (ajithranka) [inline components/OverviewSection.vue]: Do we need this wrapping `div`?

**Review by ajithranka — COMMENTED** [review]
The mobile view is a bit difficult to read because of the text column getting too narrow. Should we reduce the icon and text size, or request a design change to a single column?

https://user-images.githubusercontent.com/11095038/170555497-e0eeab8d-7bb9-41ee-a6e0-1687ce0a5527.mov

**Comment by shrinathprabhu** [comment]
> The mobile view is a bit difficult to read because of the text column getting too narrow. Should we reduce the icon and text size, or request a design change to a single column?
> 
>  RPReplay_Final1653590373.2.mov

@ajithranka check now and let me know. Added different sizing for medium screen and small screen

**Review by ajithranka — APPROVED** [review]
Looks more readable now, thanks @shrinathprabhu.

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**